### ISSUE TITLE
Input video device as source

### ISSUE THREAD

**Issue by john-bonachon** [mapmapteam/mapmap#386]
Is it possible to select an input video device as video source? Is it on MapMap plans?

**Comment by sofian** [comment]
Which version are you using? It might only be working on the develop branch.

**Comment by sofian** [comment]
Yes, the camera is not available in any of the releases yet but stay tuned for our next release which will come up mid-april.
